reframe/core/pipeline.py

```diff
@@ -1623,8 +1623,7 @@ def check_performance(self):
                     sn.evaluate(
                         sn.assert_reference(
                             val, ref, low_thres, high_thres,
-                            msg=('failed to meet reference: %s={0}, '
-                                 'expected {1} (l={2}, u={3})' % tag))
+                        )
```

Comment:
Uhm, I just realised the `tag` isn't passed anymore, so there is no way to read the name of the failing variable.  This looks tricky since the logic for greater/less than is only in  `reframe.utility.sanity.assert_reference` :confused: